public/app/plugins/panel/pluginlist/PluginList.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { GrafanaTheme, PanelProps, PluginMeta, PluginType } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { useAsync } from 'react-use';
+import { getBackendSrv } from 'app/core/services/backend_srv';
+import { UserDTO } from 'app/types';
+import { CustomScrollbar, ModalsController, stylesFactory, Tooltip, useStyles } from '@grafana/ui';
+import { UpdatePluginModal } from './components/UpdatePluginModal';
+import UserProvider from 'app/core/utils/UserProvider';
+import { css, cx } from 'emotion';
+
+export function PluginList(props: PanelProps) {
+  const pluginState = useAsync(async () => {
+    const plugins: PluginMeta[] = await getBackendSrv().get('api/plugins', { embedded: 0, core: 0 });
+    return [
+      { header: 'Installed Apps', list: plugins.filter((p) => p.type === PluginType.app), type: PluginType.app },
+      { header: 'Installed Panels', list: plugins.filter((p) => p.type === PluginType.panel), type: PluginType.panel },
+      {
+        header: 'Installed Datasources',
+        list: plugins.filter((p) => p.type === PluginType.datasource),
+        type: PluginType.datasource,
+      },
+    ];
+  }, []);
+
+  const styles = useStyles(getStyles);
+
+  if (pluginState.loading || pluginState.value === undefined) {
+    return null;
+  }
+
+  return (
+    <CustomScrollbar autoHeightMin="100%" autoHeightMax="100%">
+      <div className={styles.pluginList}>
+        {pluginState.value.map((category) => (
+          <div className={styles.section} key={`category-${category.type}`}>
+            <h6 className={styles.sectionHeader}>{category.header}</h6>
+            {category.list.map((plugin) => (
+              <a className={styles.item} href={plugin.defaultNavUrl} key={`plugin-${plugin.id}`}>
+                <img src={plugin.info.logos.small} className={styles.image} width="17" height="17" alt="" />
```

Comment:
```suggestion
                <img src={plugin.info.logos.small} className={styles.image} alt={`${plugin.name} logo`} />
```
It's a good idea to always provide a descriptive `alt` attribute to images, from the accessibility point of view. Also instead of setting dimensions as attributes, it's better to set them in `styles.image`, which makes it easier to override and tweak later.

Reply:
Re: empty alt text, this is an accepted practice for purely decorative elements, as a logo doesn't provide any additional semantics to a visually-impaired user. Re: using width/height html properties, it's generally recommended to use these in addition to css properties, as it helps prevent layout reflow while the image loads in.